core/runtime/bindings.rs

```diff
@@ -272,7 +310,41 @@ pub(crate) fn initialize_deno_core_ops_bindings<'s>(
   );
 
   let undefined = v8::undefined(scope);
-  for op_ctx in op_ctxs {
+
+  let deno_core_op_ctxs = &op_ctxs[..last_deno_core_op_id];
+  let ext_op_ctxs = &op_ctxs[last_deno_core_op_id..];
```

Comment:
I'm a bit afraid that we might be complicating it even more.

Pardon my ignorance, why do we need to seperate `core.coreOps` from `core.ops`?

Reply:
This is just first step in a bigger refactor - ops defined in `deno_core` should not be accessible outside of `deno_core` itself - if they need to be, they should be exposed as methods on `Deno.core` namespace. This temporary `Deno.core.coreOps` is only available for the short duration of `01_core.js` execution. Then it's removed immediately. These ops are necessary for the runtime to function properly.

That way, `Deno.core.ops` will only contain ops defined in extensions - then again, we want to move away from using `Deno.core.ops` altogether and instead import them directly from `ext:core/ops` (though I'm still facing some issues that I haven't figured out how to tackle). 

After some back and forth I decided that this is the right way and it requires some intermediate refactors - I want to split `op_ctxs` into two separate vectors for "core" and "ext" ops.

Reply:
Thanks for explaining, sounds good.